pyscf/neo/ase.py

```diff
@@ -86,52 +101,85 @@ def calculate(self, atoms=None, properties=['energy', 'forces', 'dipole'],
                   nuc_basis=self.parameters.nuc_basis,
                   charge=self.parameters.charge,
                   spin=self.parameters.spin)
-        if self.parameters.spin == 0:
-            mf = neo.CDFT(mol, epc=self.parameters.epc)
+
+        # If there is no mf, create one
+        if self.mf is None:
+            if self.parameters.spin == 0:
+                mf = neo.CDFT(mol, epc=self.parameters.epc)
+            else:
+                mf = neo.CDFT(mol, unrestricted = True)
```

Comment:
See https://github.com/theorychemyang/pyscf/blob/28e8f5cb910f31444b9018e30131c7e0f0113d3a/pyscf/neo/ase.py#L92 You are copying code from old version of this file.